client/src/components/TestSuite/TestSuiteDetails/styles.tsx

```diff
@@ -100,6 +100,8 @@ export default makeStyles((theme: Theme) => ({
   alert: {
     marginBottom: '8px',
     alignItems: 'center',
+    printColorAdjust: 'exact',
+    WebkitPrintColorAdjust: 'exact',
```

Comment:
TIL.  In case anyone else was wondering, this is supported on recent versions of major browsers https://caniuse.com/?search=print-color-adjust